ocpmodels/common/transforms.py

```diff
@@ -60,13 +61,23 @@ def __call__(self, data):
         # LinearTransformation only rotates `.pos`; need to rotate `.cell` too.
         data_rotated = LinearTransformation(matrix)(data)
         if hasattr(data_rotated, "cell"):
-            data_rotated.cell = torch.matmul(data_rotated.cell, matrix)
+            if torch_geometric.__version__.startswith("2.0"):
+                data_rotated.cell = torch.matmul(data_rotated.cell, matrix.T)
```

Comment:
Looks good! Rather than the extra lines of code can we just do `matrix = matrix.T` for this condition and leave the rest the same?

Reply:
Yep, makes sense